### PR TITLE
Adding editor support for expressions

### PR DESCRIPTION
**Pull request by Drifftr**

### CODE REVIEW

File: plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java [hunk 1]
```diff
@@ -138,37 +139,61 @@ private void doFinish(String containerName, String fileName, String taskName, St
...
         final IFile htconfigfile = container.getFile(new Path(HumantaskEditorConstants.INITIAL_HTCONFIG_NAME));
         addNature(container.getProject());
-        IResource [] memberList = container.members();
-        for(IResource member : memberList){
-            IFile memberFile = (IFile)member;
-            if(memberFile.getFileExtension().equals("ht")){
+        IResource[] memberList = container.members();
+        for (IResource member : memberList) {
+            IFile memberFile = (IFile) member;
+            if (memberFile.getFileExtension().equals("ht")) {
```
Comment (himasha) [inline plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java]: Add extension as a constant.

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/25

File: plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js [hunk 2]
```diff
@@ -922,6 +922,31 @@ function addPeopleAssignementNode(taskNode, xmlDom, assignmentName) {
...
 /*
+ * Signature: addExpressionNode(taskNode, xmlDom, assignmentName)  {...}
+ * 
+ * This function adds a people assignment expression node to the relevant taskNode
+ * 
+ */
+function addExpressionNode(taskNode, xmlDom, assignmentName) {
+    try {
+        if(taskNode.getElementsByTagName("peopleAssignments")[0].getElementsByTagName(assignmentName).length != 0)
+        taskNode.getElementsByTagName("peopleAssignments")[0].removeChild(taskNode.getElementsByTagName("peopleAssignments")[0].getElementsByTagName(assignmentName)[0]);
```
Comment (nandika) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: introduce {} for the if statement for clarity

Comment (Drifftr) [inline plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/hteditor.js]: Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/29

**Pull request merged**